src/MatchingResult.php

```diff
@@ -86,6 +72,9 @@ public function methods(): array
         return $this->methods;
     }
 
+    /**
+     * @psalm-assert-if-true !null $this->route
```

Comment:
Why is it needed?

Reply:
Without assertion `PossiblyNullReference` psalm error occurs.

Reply:
It shouldn't because there is check for null in the function body 😕